src/main/java/org/trebol/integration/mailing/mailgun/MailgunMailingServiceImpl.java

```diff
@@ -59,6 +59,9 @@
 public class MailgunMailingServiceImpl
   implements IMailingIntegrationService {
 
+  private final static String OWNERS_MAPS_KEY_PREFIX = "owners:";
+  private final static String CUSTOMER_MAPS_KEY_PREFIX = "customer:";
```

Comment:
The modifiers needs to be in the correct order. The modifier "static" comes before the "final".

Source: https://sonarcloud.io/organizations/trebol-ecommerce/rules?open=java%3AS1124&rule_key=java%3AS1124

Reply:
Done, thank you